internal/services/redis/redis_cache_resource_test.go

```diff
@@ -208,7 +208,7 @@ func TestAccRedisCache_BackupEnabledDisabled(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_redis_cache", "test")
 	r := RedisCacheResource{}
 
-	data.ResourceTest(t, r, []acceptance.TestStep{
+	data.ResourceTestIgnoreRecreate(t, r, []acceptance.TestStep{
```

Comment:
how come we are ignoring the recreation for this test? it doesn't appear the backup properties force a new resource?